cc/defs.bzl

```diff
@@ -28,6 +28,9 @@ BINARY = "binary"
 # Name for swift_cc_test_library
 TEST_LIBRARY = "test_library"
 
+# Name for swift_cc_test
+TEST = "test"
```

Comment:
What was this necessary for?

Reply:
I added it in order to nicely build all targets except test and test_library:
`bazel build //... --build_tag_filters="-test,-test_library"`